dist/sdk.js

```diff
@@ -686,6 +686,24 @@
             });
         };
 
+        /**
+         * Returns flag status on account
+         * @param {string} flagName
+         * @param {function} [onSuccess]
+         * @param {function} [onError]
+         * @returns {number} callId
+         */
+        this.hasFeatureFlag = function (flagName, onSuccess, onError) {
+            return this.method({
```

Comment:
`this.validate` validation missing - see this.deleteTheme method for example

Reply:
good point. fixed